### DISCUSSION TITLE
Exposing library version

### DISCUSSION THREAD

**Discussion by eduardorn** [pdfcpu/pdfcpu#977] [Q&A] [answered]
Is there any way we can disable the exposure of the following?

- Version: "pdfcpu " + model.Version
- Creator
- CreationDate
- Producer

**Comment by hhrutter** [comment]
Please explain the details of your usecase and the issue so I understand what you are referring to.
Thank you!

**Reply by eduardorn** [reply]
At some point in our process, when we call MergeCreateFile(), and as part of the response, we are getting something like this:

```
<<CreationDate...../ModDate.....Producer(pdfcpuv0.4.1 dev)>>
```

According to security best practices, exposing the library version might be a risk, so we want to avoid this line being exposed.

**Comment by hhrutter** [comment, marked as answer]
I am still not exactly sure what you mean and what the problem is.
Security??

When writing back a file pdfcpu records itself as the producer in the info dict - common practice!

When you open the file with a viewer just go to document properties and this usually shows up.

**Reply by eduardorn** [reply]
Ok. Thank you for the explanation @hhrutter.
